modules/virt-configuring-disk-sharing-lun.adoc

```diff
@@ -6,11 +6,12 @@
 [id="virt-configuring-disk-sharing-lun_{context}"]
 = Configuring disk sharing by using LUN
 
-You can configure a LUN-backed virtual machine (VM) disk to be shared among multiple virtual machines by enabling SCSI persistent reservation. Enabling the shared option allows you to use advanced SCSI commands, such as those required for a Windows failover clustering implementation, against the underlying storage. Any disk to be shared must be in block mode.
+To protect data on your VM from outside access, you can enable SCSI persistent reservation and configure a LUN-backed virtual machine disk to be shared among multiple virtual machines. Enabling the shared option allows you to use advanced SCSI commands, such as those required for a Windows failover clustering implementation, for managing the underlying storage.
```

Comment:
Per ISG,

> Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”.

```suggestion
To secure data on your VM from unauthorized access, you can enable SCSI persistent reservation and configure a LUN-backed virtual machine disk to be shared among multiple virtual machines. By enabling the shared option, you can use advanced SCSI commands, such as those required for a Windows failover clustering implementation, for managing the underlying storage.
```

Reply:
I'd say that "enabling option X allows you to do Y" is not really an instance of anthropomorphism, but your suggested rewrite definitely is an improvement. Thanks Shubha!